RELEASE.md

```diff
@@ -6,6 +6,18 @@
 - [GNU Make](https://www.gnu.org/software/make/) `v4.x`
 - [Kustomize](https://github.com/kubernetes-sigs/kustomize) `v1.3.x`
 
+## Github Workflow Test Matrix Checkup
```

Comment:
Now I've realized that maybe this should be an item under "Documentation" instead of a section on its own? This only makes sense after a new directory for the new major/minor version gets created - probably also in the same PR as the rest of the documentation changes.

Or, at the very least, have the reader perform this only after the "documentation" section is taken care of.

Reply:
I'm not sure I'm getting what you're saying. To me it seems because we don't yet have any automation for this, we should be checking this before _any_ release, that is the entire purpose of the PR?

Reply:
Let's try.